### ISSUE TITLE
Improve Logging

### ISSUE THREAD

**Issue by egucciar** [julianburr/cypress-match-screenshot#5]
Currently the logging is rather unhelpful. Would be nice if it could be improved i.e. including the name of the screenshot being taken per step.

**Comment by julianburr** [comment]
Yes, the logging def needs to be improved, will look into that in the next days ... if you have any other suggestions for helpful logs besides the file names feel free to shoot them through :blush:
